content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -194,6 +194,68 @@ For example, if there are 1,253 pods on the cluster and the client wants to rece
 
 Note that the `resourceVersion` of the list remains constant across each request, indicating the server is showing us a consistent snapshot of the pods. Pods that are created, updated, or deleted after version `10245` would not be shown unless the user makes a list request without the `continue` token.  This allows clients to break large requests into smaller chunks and then perform a watch operation on the full set without missing any updates.
 
+## Lists
+
+There are dozens of list types (such as `PodList`, `ServiceList`, and `NodeList`) defined in the Kubernetes API.
+You can get more information about each list type from the [Kubernetes API](https://kubernetes.io/docs/reference/kubernetes-api/) documentation.
+
+When you query the API for a particular type, all items returned by that query are of that type. For example, when you
+ask for a list of pods, the list type is shown as `kind: PodList` and each `item` in that list represents a pod. For example:
+
+```console
+
+GET /api/v1/namespaces/kube-system/pods
+---
+{
+  "kind": "PodList",
+  "apiVersion": "v1",
+  "metadata": {
+    "resourceVersion": "2873495"
+  },
+  "items": [
+    {
+      "metadata": {
+        "name": "coredns-558bd4d5db-hfgxg",
+        "generateName": "coredns-558bd4d5db-",
+...
+
+      "metadata": {
+        "name": "weave-net-rnbqq",
+        "generateName": "weave-net-",
+...
+```
+
+Some tools, such as `kubectl` provide another way to query the Kubernetes API. Because the output of `kubectl` might include multiple list types, the list of items is represented as `kind: List`. For example:
+
+
+```console
+
+$ kubectl get pods -n kube-system -o yaml 
+
+apiVersion: v1
+items:
+- apiVersion: v1
+ kind: Pod
+  metadata:
+kind: Pod
```

Comment:
Is this right? The indentation looks wrong.

Reply:
It might be easier to fetch a list of smaller objects. Maybe Lease or ServiceAccount?

Reply:
Good catch! Fixed. Maybe you are right. I'll try some smaller examples.

Reply:
@sftim I tried a few different ones and changes the examples to use Service. I showed the same result querying the api directly and through kubectl. Let me know what you think.